src/components/CampaignTextersForm.jsx

```diff
@@ -279,13 +279,12 @@ export default class CampaignTextersForm extends React.Component {
       .map(orgTexter => dataSourceItem(orgTexter.displayName, orgTexter.id));
 
     const filter = (searchText, key) =>
-      key === "allTexters"
-        ? true
-        : AutoComplete.caseInsensitiveFilter(searchText, key);
+      key === "allTexters" ? true : AutoComplete.fuzzyFilter(searchText, key);
 
     const autocomplete = (
       <AutoComplete
         ref="autocomplete"
+        s
```

Comment:
is this a typo or a real property?

Reply:
Good catch @schuyler1d  -- fixed the typo